package.json

```diff
@@ -32,8 +33,9 @@
     "karma-coverage": "^1.0.0",
     "karma-jasmine": "^0.3.6",
     "karma-phantomjs-launcher": "^0.2.1",
+    "moment": "^2.17.1",
     "phantomjs": "^1.9.18",
-    "protractor": "^3.1.1",
+    "protractor": "^4.0.8",
```

Comment:
Yes @ebragaparah it's issue now for me. I can't run the e2e locally I need start webdriver  manually before run. Maybe we can create the new story to resolve this problem. What you think?